packages/utils/run-prettier.ts

```diff
@@ -19,18 +19,17 @@ export default function runPrettier(outputPath: string, source: string, cb?: Fun
 		try {
 			prettySource = prettier.format(source, {
 				filepath: outputPath,
-				parser: "babylon",
+				parser: "babel",
```

Comment:
`jscodeshift` makes transforms using `babylon` parser currently. This may give inconsistent results.

Reply:
Why would it? Here we just format a string using prettier (with babel, because babylon is deprecated), I don't any `jscodeshift` thing that could cause troubles

Reply:
I wasn't sure @ematipico wrote this review because, prettier will be formating the source string that also includes the transforms made using babylon parser and as babel is quiet new as compared to babylon and may not match the some of the old syntactical structures of commonJS ( if there's any ).

But still I may be wrong here :)

Reply:
Looking at `jscodeshift`'s [README](https://www.npmjs.com/package/jscodeshift) seems that it uses `babel` as default parser

```
 --parser=babel|babylon|flow|ts|tsx  the parser to use for parsing the source files
                                          (default: babel)
```

Searching in `webpack-cli` source code I can't find any `babylon` occurence, so I guess we are not overriding that default value.
What do you think @rishabh3112? :)

Reply:
We are using an older version of jscodeshift (0.5.x) which uses babylon for parsing.

Reply:
0.5.1 to be precise , refer utils package:)

Reply:
Fair point @rishabh3112 	
I wouldn't too much though, babel and babylon are the same parser at the end. I am optimistic and I'd say it won't break things 😄

Reply:
Ya sure, it was tinkling me at back of my mind 😂

Reply:
Need to change it to babylon as tsc is complaining